engine/release-notes/index.md

```diff
@@ -15,6 +15,38 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.23
+{% include release-date.html date="2023-01-19" %}
+
+This release of Docker Engine contains updated versions of Docker Compose,
+Docker Buildx, Containerd, and some minor bug fixes and enhancements.
+
+### Updates
+
+- Update Docker Compose to [v2.15.1](https://github.com/docker/compose/releases/tag/v2.15.1){:target="_blank" rel="noopener"}.
+- Update Docker Buildx to [v0.10.0](https://github.com/docker/buildx/releases/tag/v0.10.0){:target="_blank" rel="noopener"}.
+- Update containerd (`containerd.io` package) to [v1.6.15](https://github.com/containerd/containerd/releases/tag/v1.6.15){:target="_blank" rel="noopener"}.
+- Update the package versioning format for `docker-compose-cli` to allow distro version updates [docker/docker-ce-packaging#822](https://github.com/docker/docker-ce-packaging/pull/822){:target="_blank" rel="noopener"}.
+- Update Go runtime to [1.18.10](https://go.dev/doc/devel/release#go1.18.minor){:target="_blank" rel="noopener"},
+
+### Bug fixes and enhancements
+
+- Fix an issue where `docker build` would fail when using `--add-host=host.docker.internal:host-gateway`
+  with BuildKit enabled [moby/moby#44650](https://github.com/moby/moby/pull/44650){:target="_blank" rel="noopener"}.
+- Revert seccomp: block socket calls to `AF_VSOCK` in default profile [moby/moby#44712](https://github.com/moby/moby/pull/44712){:target="_blank" rel="noopener"}.
+  This change, while favorable from a security standpoint, caused a change
```

Comment:
Suggest to add a newline here to break the paragraph

```suggestion

  This change, while favorable from a security standpoint, caused a change
```

Reply:
@dvdksn updated; ptal 👍